lib/puppet/transport/cisco_ios.rb

```diff
@@ -380,9 +380,9 @@ def run_command_acl_mode(acl_name, acl_type, command)
       send_command(connection, 'exit', true)
     end
 
-    def running_config_save(dest = 'startup-config')
+    def save_config(from: 'running-config', to: 'startup-config')
```

Comment:
Since you're already renaming this function, I'd remove a layer of confusion here and call it the same as the underlying CLI command. Make this `copy(from:, to:)` and also call the new task `copy` with the matching arguments. This will make it easier for folks comfortable with the CLI to pick this up.